src/modules/db/mongodb-init.ts

```diff
@@ -6,13 +6,13 @@ import * as log from '../../utils/log';
 // mongoose.set('useUnifiedTopology', true);
 // mongoose.set('useCreateIndex', true);
 
-const prepareConnectionString = (config: {
+export const prepareConnectionString = (config: {
```

Comment:
I wanted to test this function but had to export it to make it working. Was that proper procedure or would there be a better way to do it? @jimmyhogoboom

Reply:
Afaik, that's the only way. It does bring up the argument about whether that function should be tested if it isn't part of the "public" interface of the module. I don't really see the harm in it though.